0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md

```diff
@@ -33,108 +31,90 @@ The sending and receiving ledger are different ledgers. Otherwise, there should
 
 ## Get Quote
 
-Quotes are requested with `plugin.sendMessage(message)` and returned via `plugin.on("incoming_message")` (see the Ledger Plugin Interface for more details).
-The message formats described here are the `data` properties of an `OutgoingMessage`.
+Quotes are requested with `plugin.sendRequest(message)`, which returns the quote response (see the Ledger Plugin Interface for more details).
```

Comment:
also, if wouldn't the equivalent of `sendMessage(message)` be `sendRequest(request)` or  `sendRequest(quoteRequest)`?